packages/project-account-switcher/src/ProjectAccountSwitcher.js

```diff
@@ -1,6 +1,95 @@
-import ProjectAccountSwitcher from "hig-react/lib/elements/components/GlobalNav/TopNav/ProjectAccountSwitcher";
-import "./project-account-switcher.scss";
+import React, { Component } from "react";
+import PropTypes from "prop-types";
 
-ProjectAccountSwitcher.displayName = "ProjectAccountSwitcher";
+import ProjectAccountSwitcherPresenter from "./presenters/ProjectAccountSwitcherPresenter";
+import ProjectAccountSwitcherBehavior from "./behavior/ProjectAccountSwitcherBehavior";
 
-export default ProjectAccountSwitcher;
+/* eslint no-shadow: "off" */
+
+export default class ProjectAccountSwitcher extends Component {
+  static propTypes = {
+    /** Heading title for the list of Accounts */
+    accountTitle: PropTypes.string,
+    /** List of Accounts */
+    accounts: PropTypes.arrayOf(
+      PropTypes.shape({
+        id: PropTypes.string,
+        image: PropTypes.string,
+        label: PropTypes.string
+      })
+    ),
+    /** Currently selected Account */
+    activeAccountId: PropTypes.string,
+    /** Currently selected Project */
+    activeProjectId: PropTypes.string,
+    /** Called when a the element changes */
+    onChange: PropTypes.func,
+    /** Called when a user clicks on the element */
+    onClick: PropTypes.func,
+    /** Called when a user clicks on the target element */
+    onTargetClick: PropTypes.func,
+    /** List of Projects */
+    projects: PropTypes.arrayOf(
+      PropTypes.shape({
+        id: PropTypes.string,
+        image: PropTypes.string,
+        label: PropTypes.string
+      })
+    ),
+    /** Heading title for the list of Projects */
+    projectTitle: PropTypes.string
+  };
+
+  state = {
```

Comment:
is this being used?

Reply:
Hm! I thought it was but I think you are right. ✅